### PR TITLE
Update Swiss-french (fr-CH) locale

### PR DESCRIPTION
**Pull request by DCBN**
This PR updates the Swiss-french locale to enable the locale in date-fns.

### CODE REVIEW

File: src/locale/fr-CH/_lib/formatRelative/index.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-  yesterday: '[hier à] LT',
-  today: '[aujourd’hui à] LT',
-  tomorrow: '[demain à] LT',
-  nextWeek: 'dddd [à] LT',
-  other: 'L'
+  lastWeek: "eeee 'la semaine dernière à' p",
+  yesterday: "'hier à' p",
+  today: "'aujourd’hui à' p",
+  tomorrow: "'demain à' p'",
+  nextWeek: "eeee 'la prochaine dernière à' p",
```
[the remark this answers is not in the document]
Comment (schwindel) [reply src/locale/fr-CH/_lib/formatRelative/index.js:6]: To be precise `"eeee 'la semaine prochaine à' p"`

File: src/locale/fr-CH/_lib/formatLong/index.js [hunk 2]
```diff
@@ -1,12 +1,41 @@
...
-  L: 'DD.MM.YYYY',
-  LL: 'Do MMMM YYYY',
-  LLL: 'Do MMMM YYYY HH:mm',
-  LLLL: 'dddd, Do MMMM YYYY HH:mm'
-})
+var dateFormats = {
+  full: 'EEEE d MMMM y',
+  long: 'd MMMM y',
+  medium: 'd MMM y',
+  short: 'dd/MM/y'
```
[the remark this answers is not in the document]
Comment (DCBN) [reply src/locale/fr-CH/_lib/formatLong/index.js:7]: I will update this!

File: src/locale/fr-CH/_lib/formatRelative/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (DCBN) [reply src/locale/fr-CH/_lib/formatRelative/index.js:6]: I can fix this too!

**Comment by DCBN** [comment]
Fixed both of the requested changes by @schwindel

**Pull request merged**